microsvcs/forum/app/controller/comments_con.js

```diff
@@ -1,55 +1,52 @@
 const QUERIES = require("../model/comments_model.js")
 const FORUM_QUERIES = require("../model/forum_model.js")
-const HANDLER = require("../util/handler.js")
+const UTIL = require('../util/init_res.js')
+
 /**
  * Add a comment
  * 
  * @param {String} req.params.uid user id, unique uid field in db 
  * @param {String} req.params.post_id post_id - unique _id field in each document
  * @param {String} req.body.comment user comment in plaintext
  */
-exports.addComment = async (req, res) => {
+exports.addComment = async (req, res, next) => {
+    const post_id = req.params.post_id;
+    const uid = req.body.uid;
+    const comment = req.body.comment;
     try {
-        const post_id = req.params.post_id;
-        const uid = req.body.uid;
-        const comment = req.body.comment;
-        // Check if the post exists
         const check = await FORUM_QUERIES.getPostbyID(post_id)
         if (!check) {
-            throw new Error("Reference Not Found")
+            next(UTIL.formatRes(false, 404, "No Reference Found"))
         }
         const result = await QUERIES.addComment(post_id, uid, comment)
         if (result) {
-            res.status(200).send(HANDLER.createSuccessResponse("Comment Added", { "comment_id": result }));
+            next(UTIL.formatRes(true, 200, "Comment Added", { "comment_id": result }))
         }
     } catch (error) {
-        if (error.message == "Server Error Occurred") {
-            res.status(500).send(HANDLER.createErrorResponse(error.message));
-        } else {
-            res.status(404).send(HANDLER.createErrorResponse(error.message));
-        }
+        next(UTIL.formatRes(false, 500, "Database Error"))
     };
 };
 
 /**
  * Delete a comment
  * 
  * @param {String} req.body.uid unique uid field in db 
- * @param {String} req.body.post_id unique post id
+ * @param {String} req.params.post_id unique post id
  * @param {String} req.body.comment_id unique comment id
  */
-exports.deleteComment = async (req, res) => {
+exports.deleteComment = async (req, res, next) => {
+    const uid = req.body.uid;
+    const post_id = req.params.post_id;
+    const comment_id = req.body.comment_id;
     try {
-        const uid = req.body.uid;
-        const post_id = req.params.post_id;
-        const comment_id = req.body.comment_id;
         const result = await QUERIES.deleteComment(post_id, uid, comment_id)
-        res.status(200).send(HANDLER.createSuccessResponse("Comment Deleted", result));
-    } catch (error) {
-        if (error.message == "Server Error Occurred") {
-            res.status(500).send(HANDLER.createErrorResponse(error.message));
+        if (result.deletedCount === 1) {
+            next(UTIL.formatRes(true, 200, "Comment Deleted"));
         } else {
-            res.status(404).send(HANDLER.createErrorResponse(error.message));
+            next(UTIL.formatRes(false, 404, "No Reference Found"))
         }
+    } catch (error) {
+        console.log(error)
```

Comment:
you can consider exploring better logging options for e.g. `express-winston`, which is a popular logging library for express